agent/php_txn.c

```diff
@@ -625,17 +625,17 @@ static void nr_php_txn_log_error_dt_on_tt_off(void) {
 }
 
 static void nr_php_txn_send_metrics_once(nrtxn_t* txn TSRMLS_DC) {
-  static unsigned int sent = 0;
   char* metname = NULL;
 
   if (nrunlikely(NULL == NRPRG(txn))) {
     return;
   }
 
-  if (nrlikely(0 != sent)) {
+  if (nrlikely(0 != txn->created_logging_onetime_metrics)) {
     return;
   }
 
+  nrl_verbosedebug(NRL_TXN, "creating one time logging metrics");
```

Comment:
Can probably remove this debug line

Reply:
Yeah it should only happen once or twice during a request and is only when debugging level is active but I'll remove it.